x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -136,11 +148,17 @@ export interface ActivityLogActionResponse {
     data: EndpointActionResponse;
   };
 }
+
+/**
+ * One of the possible Response Action Log entry - Eitehr a Fleet Action request, Fleet action response,
```

Comment:
Either

Reply:
Speaking of that, While we may not have 7.14 Endpoints any more, sending actions in the old format. We _may_ still have the data sitting around from those things. Are we okay for that old action data to show up in a weird state if we just move on to supporting only the new action format?

Reply:
Fixed the typo (thanks Dan).

I'm not sure about the other comments, but I'm wondering - does it matter? Looking at the defintion, the `types` only has two values (fleet or endpoint).